test/index/namespaces/namespaces.snapshot.cc

```diff
@@ -46,22 +46,22 @@
   #define EXPAND_TO_NAMESPACE \
 //        ^^^^^^^^^^^^^^^^^^^ definition [..] namespaces.cc:36:9#
     namespace from_macro {}
-//            ^^^^^^^^^^ definition [..] from_macro/
-//            ^^^^^^^^^^ definition [..] from_macro/
   
   EXPAND_TO_NAMESPACE
 //^^^^^^^^^^^^^^^^^^^ reference [..] namespaces.cc:36:9#
+//^^^^^^^^^^^^^^^^^^^ definition [..] from_macro/
   
   #define EXPAND_TO_NAMESPACE_2 EXPAND_TO_NAMESPACE
 //        ^^^^^^^^^^^^^^^^^^^^^ definition [..] namespaces.cc:41:9#
 //                              ^^^^^^^^^^^^^^^^^^^ reference [..] namespaces.cc:36:9#
   
   EXPAND_TO_NAMESPACE_2
 //^^^^^^^^^^^^^^^^^^^^^ reference [..] namespaces.cc:41:9#
+//^^^^^^^^^^^^^^^^^^^^^ definition [..] from_macro/
   
   #define IDENTITY(x) x
 //        ^^^^^^^^ definition [..] namespaces.cc:45:9#
   
   IDENTITY(namespace in_macro { })
 //^^^^^^^^ reference [..] namespaces.cc:45:9#
-//                   ^^^^^^^^ definition [..] in_macro/
```

Comment:
This one looks better for "in_macro", curious why this changed.

Reply:
This changed because we're using the expansion location instead of the spelling location. The expansion location uniformly handles all cases, but the spelling location needs some fine-tuning because it's not always available. I've chosen to prioritize availability in this PR to keep moving.